module4-authentication-and-security/r1.1.1-authentication-persistence/README.md

```diff
@@ -0,0 +1,268 @@
+# Authentication Persistence
+
+In this lesson, we will tackle how to persist user login, or registration between requests. Learning objectives are:
+
+- Difference between stateful and stateless persistence
+- Implementing stateful persistence using express-sessions
+- Implementing stateless identifiers using JSON Web Tokens (JWT)
+
+## **Stateful Session Storage**
+
+In web servers, sessions are server-side temporary storage tools. They are used for a variety of use cases to store temporary data between requests. They use different storage adapters, like RAM, disk, databases, in-memory cache, etc. to save the temporary data.
```

Comment:
i don't understand these two paragraphs, i would consider rereading them from the perspective of a student 

i think you should explain what you mean by a session first, at least, using concrete examples

after that the distinction between persistent and nonpersistent needs to be more clear